app/res/xml/preferences_developer.xml

```diff
@@ -85,6 +85,11 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-auto-purge"
         android:title="Detect and Trigger Purge on Form Save"/>
+    <EditTextPreference
```

Comment:
Ha, right, I forgot that key point. I still think it's important to make this more usable in this way though, and actually don't think it will be a problem in calabash from thinking a bit more about it. If you look at how the step for `"I Install the .ccz file at {some_file_path}"` is implemented in our calabash steps, that also relies on the behavior of selecting a file from the file system, and that has been quite reliable thus far.